apitest/src/main/java/bisq/apitest/scenario/FundWalletScenarioTest.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * This file is part of Bisq.
+ *
+ * Bisq is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or (at
+ * your option) any later version.
+ *
+ * Bisq is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public
+ * License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with Bisq. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package bisq.apitest.scenario;
+
+import lombok.extern.slf4j.Slf4j;
+
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static java.lang.Double.parseDouble;
+import static java.util.concurrent.TimeUnit.MILLISECONDS;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+
+
+import bisq.apitest.OrderedRunner;
+import bisq.apitest.annotation.Order;
+
+@Slf4j
+@RunWith(OrderedRunner.class)
+public class FundWalletScenarioTest extends ScenarioTest {
+
+    @BeforeClass
+    public static void setUp() {
+        try {
+            setUpScaffold("bitcoind,seednode,alicedaemon");
+            bitcoinCli.generateBlocks(1);
+            MILLISECONDS.sleep(1500);
+        } catch (InterruptedException ex) {
+            fail(ex.getMessage());
+        }
+    }
+
+    @Test
+    @Order(1)
+    public void testFundWallet() {
+        long balance = getBalance();  // bisq wallet was initialized with 10 btc
+        assertEquals(1000000000, balance);
+
+        String unusedAddress = getUnusedBtcAddress();
+
+        // Given the default tx fee rate, we want to send 2.5 + the fee,
+        // so the new wallet balance will be exactly 12.5 btc.
+        // We should calculate the fee based on the fee rate and tx size
+        // instead of hard coding the fee amount.
+        double btc = parseDouble("2.5") + parseDouble("0.0000336");
+        bitcoinCli.sendToAddress(unusedAddress, btc);
+
+        bitcoinCli.generateBlocks(1);
+        sleep(1500);
```

Comment:
`bitcoind` has the `blocknotify` callback for this. Not really a problem now, but something to consider if the number of "sleeps" grows cumbersome.

Reply:
This will become a nuisance, but right now I don't know of another way to give `:daemon` some time to parse new blocks.  (The callback does not reach the `:cli`.)